### ISSUE TITLE
Cannot make bootloader file to make .coe file for AEE ROM

### ISSUE THREAD

**Issue by HollaHieu** [skordal/potato#24]
Hi skordal! I really appreciate your work on risc v potato processor. I am having an issue setting up and running the potato processor. I am trying to follow your System-On-Chip Design using the Potato Processor using vivado 2020.3 . The problem I encounter is whenver I try to "make" the bootloader file in windows power shell, I always run into the problem (described in the pictures) and cannot output .coe file. Can you shed any light on what I have done wrong? \
![ảnh](https://user-images.githubusercontent.com/43841523/123036712-b15e0f80-d417-11eb-87ff-438ff6b638f5.png)

**Comment by skordal** [comment]
Hi,

It looks like some file cannot be found, is riscv32-unknown-elf-gcc accessible from the command line? 

I have only tested the makefiles with GNU make on Linux systems, so there might be trouble when attempting to get them to run on Windows.

**Comment by HollaHieu** [comment]
you're right risc32-unknown-elf-gcc is not available from the command line. I can try install risc-v GNU compiler toolchain, which I supposed will include the mentioned file. However, GNU compiler toolchain is also easily downloadable in Linux systems. So, I should probably move to an Ubuntu for this project.  Thank you mate!

**Comment by HollaHieu** [comment]
Hi @skordal , If you have some free time, may I ask you for the .coe (coefficient file) for the bootloader application to be included in the repository code? I am having trouble getting that file only.
